common-api/src/test/java/org/pmiops/workbench/db/model/CommonStorageEnumsTest.java

```diff
@@ -4,103 +4,86 @@
 import static com.google.common.truth.Truth.assertWithMessage;
 
 import com.google.common.collect.BiMap;
-import java.lang.reflect.Field;
 import java.lang.reflect.Method;
 import java.lang.reflect.ParameterizedType;
-import java.util.function.Function;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
 import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameter;
-import org.junit.runners.Parameterized.Parameters;
-import org.pmiops.workbench.model.DataAccessLevel;
 import org.pmiops.workbench.model.Domain;
 
-@RunWith(Parameterized.class)
 public class CommonStorageEnumsTest {
-  @Parameters(name = "{0}")
-  public static Object[][] data() {
-    return new Object[][] {
-      {
-        DataAccessLevel.class.getSimpleName(),
-        DataAccessLevel.values(),
-        (Function<Short, DataAccessLevel>) CommonStorageEnums::dataAccessLevelFromStorage,
-        (Function<DataAccessLevel, Short>) CommonStorageEnums::dataAccessLevelToStorage
-      },
-      {
-        Domain.class.getSimpleName(),
-        Domain.values(),
-        (Function<Short, Domain>) CommonStorageEnums::domainFromStorage,
-        (Function<Domain, Short>) CommonStorageEnums::domainToStorage
-      }
-    };
-  }
-
-  @Parameter() public String description;
-
-  @Parameter(1)
-  public Enum<?>[] enumValues;
-
-  @Parameter(2)
-  public Function<Short, Enum<?>> fromStorage;
-
-  @Parameter(3)
-  public Function<Enum<?>, Short> toStorage;
-
-  @Test
-  public void testBijectiveStorageMapping() {
-    for (Enum<?> v : enumValues) {
-      Short storageValue = toStorage.apply(v);
-      assertWithMessage("unmapped enum value: " + v).that(storageValue).isNotNull();
-      assertThat(v).isEqualTo(fromStorage.apply(storageValue));
-    }
-  }
 
   // domain ID is stringly-typed so special-case this
 
   @Test
-  public void testDomainIdBijectiveStorageMapping() {
+  public void test_domainId() {
```

Comment:
How do we test the reverse direction? I.e. how do we know we don't have shorts written to the db someplace that don't map back to an enum value?

Reply:
I guess by virtue of them being in a bimap it's not a problem, but the test shouldn't know that.